### ISSUE TITLE
Global message banner commands

### ISSUE THREAD

**Issue by willsmythe** [Microsoft/vsts-cli#174]
VSTS supports the ability for account admins to present users a message at the top of every VSTS web page. This allows for communication of announcement, disruptions, or other important messages to users of the account.  

# Proposed commands

## Add banner

Add a new banner and immediately show it.

```
vsts admin banner add
```

### Required

* `-m` or `--message`. Message (string) to show in the banner.

### Optional

* `-id`. Identifier for the new banner. This identifier is needed to change or remove the message later. A unique identifier is automatically created if one is not specified.
* `-l` or `--level`. Influences the color and icon presented with the message. Valid values: `info`, `warning`, `error`
* `--expiration`. Date/time when the banner should no longer be presented to users. If not set, the banner does not automatically expire and must be removed with the `remove` command.

## Update banner

Update the message, level, or expiration date for a banner.

```
vsts admin banner update
```

### Required

* `-id`. ID of the banner to update.

### Optional

* `-m` or `--message`. New message (string) to show in the banner.
* `-l` or `--level`. Type of message (influences the color and icon presented with the message). Valid values: `info`, `warning`, `error`
* `--expiration`. Date/time when the banner should no longer be presented to users. To unset the expiration for the banner, supply an empty value to this argument.

## Remove banner

Remove (hide) the banner.

```
vsts admin banner remove
```

### Required

* `-id`. ID of the banner to remove.

## List banners

```
vsts admin banner list
```

### Optional arguments

* `--includeExpired`. Optionally include expired banners, otherwise expired banners will not be included.

# Proposed table output

* `ID` (assume IDs are GUIDs)
* `Level` (Info, Warning, Error)
* `Expiration` (formatted date/time string OR empty string; this column should only appear in the output when the `--includeExpired` argument is supplied)
* `Message` (first 40? characters)

**Comment by alexcnichols** [comment]
- Would the message allow for any kind of rich text, such as an HTML anchor tag hyperlink, or only plain text?
- You mention account admin. What's the thinking on specific permission for these commands? Would there be a specific permission or group check?

**Comment by tedchamb** [comment]
#178

**Comment by tedchamb** [comment]
Commands available in VSTS-CLI 0.1.2
